securesystemslib/interface.py

```diff
@@ -472,7 +472,7 @@ def generate_and_write_ed25519_keypair(filepath=None, password=None):
         confirm=True)
 
   else:
-    logger.debug('The password has been specified.  Not prompting for one')
+    logger.debug('The password has been specified. Not prompting for one.')
```

Comment:
Oh, you're one of those single space propagandists. :P
https://arstechnica.com/science/2018/05/two-spaces-after-period-are-better-than-one-except-maybe-they-arent-study-finds/